pages/ladybug.page.js

```diff
@@ -1,9 +1,43 @@
-var ladybugPage = function() {
-	this.stage = element.all(by.className("m-r-sm stage")).get(0);
+var LadybugPage = function() {
+	this.stage = element.all(by.className('m-r-sm stage')).get(0);
+	// sidebar components
+	this.sidebarTestingTab = element(by.className('fa fa-bug'));
```

Comment:
The sidebar components should be specified in a seperate file (like sidebar.component.js). But either way, you shouldn't need them in the ladybug tests :)